specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Create_WithAutomaticRepairs.json

```diff
@@ -59,6 +59,22 @@
         "upgradePolicy": {
           "mode": "Manual"
         },
+		"scheduledEventsPolicy": { 
+			"scheduledEventsAdditionalPublishingTargets": { 
+				"eventGridAndResourceGraph": 
```

Comment:
Sorry I gave you wrong advice. it should be like: 
"eventGridAndResourceGraph": { 
				    "enable": true 
				}, 

But to resolve all of these together, you can run prettier on them on vs code. It will format everything correctly.